x/auth/client/cli/sign.go

```diff
@@ -0,0 +1,124 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authctx "github.com/cosmos/cosmos-sdk/x/auth/client/context"
+	"github.com/spf13/cobra"
+	amino "github.com/tendermint/go-amino"
+)
+
+const (
+	flagOverwriteSigs = "overwrite"
+	flagPrintSigs     = "print-sigs"
+)
+
+// GetSignCommand returns the sign command
+func GetSignCommand(codec *amino.Codec, decoder auth.AccountDecoder) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sign <file>",
+		Short: "Sign transactions",
+		Long: `Sign transactions created with the --generate-only flag.
+Read a transaction from <file>, sign it, and print its JSON encoding.`,
```

Comment:
Can you specify accepted file formats?

Reply:
Sure, ACs and proposed specs so far focused on JSON only